pkg/kubelet/apis/deviceplugin/v1alpha/api.pb.go

```diff
@@ -135,6 +135,8 @@ type Device struct {
 	ID string `protobuf:"bytes,1,opt,name=ID,json=iD,proto3" json:"ID,omitempty"`
 	// Health of the device, can be healthy or unhealthy, see constants.go
 	Health string `protobuf:"bytes,2,opt,name=health,proto3" json:"health,omitempty"`
+	// Capabilities of the device. eg: {"version": "1.0"}
+	Capabilities map[string]string `protobuf:"bytes,3,rep,name=capabilities" json:"capabilities,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
```

Comment:
Are these changes in this file auto-generated?
If yes, LGTM.

And maybe we can add some reference in PR description about how we auto-generate these code :)

Reply:
yes, this is auto-generated. And here is how to generate: 
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/deviceplugin/v1alpha/api.proto#L1